### DISCUSSION TITLE
dependabot or not, the forks edition

### DISCUSSION THREAD

**Discussion by Pike** [JamesIves/github-pages-deploy-action#550] [General]
I've looked at branches and PRs on my fork, and it's getting messed with by dependabot.

Looking at https://github.com/JamesIves/github-pages-deploy-action/network, it's a lot of forks, actually all forks that have the `.github/dependabot.yml` file on their default branch.

I wonder if it'd be OK to switch that off until https://github.com/dependabot/dependabot-core/issues/2198 gets a fix.

**Comment by JamesIves** [comment]
That doesn't seem good. I will turn that off later today.
